docs/adr/adr_improved_frontend_dx.md

```diff
@@ -0,0 +1,58 @@
+# ADR: Improved Frontend Developer Experience
+Decision Deadline: As soon as possible
+
+Author: [Felipe](https://github.com/fhenrich33)
+
+## Status
+
+Proposed, Implementing
+
+## Context
+Frontend development experience in its current form is quite lacking. It imposes a lot of artificial difficulties, tooling doesn't work as expected and adds friction to our FE contributors.
+
+Local installation of packages and tooling might not play well with the codebase. Things like TS, ESlint, and Prettier might not work at all without a lot of fiddling. 
```

Comment:
Do @HaGuesto @vahidbazzaz @MaikNeubert currently work with the "old" FE tooling? Just curious if they experienced similar fiddling issues :sweat_smile:

Reply:
also tagging @jlin95 since she recently installed our codebase :)

Reply:
Some sorts of issues I saw:
`Cannot find module '@chakra-ui/react' or its corresponding type declarations.ts(2307)` in ui component files.
We do have the prettier and eslint config files in the codebase but it's not getting picked up properly by the IDE when you make changes that violate rules specified in those files.